Subject: Linux-RT 2.6.25-rt
From: http://www.kernel.org/pub/linux/kernel/projects/rt/
Acked-by: Sven-Thorsten Dietrich <sdietrich@suse.de>
From dwalker@mvista.com Wed Sep 26 22:18:23 2007
Date: Tue, 28 Aug 2007 14:37:52 -0700
From: Daniel Walker <dwalker@mvista.com>
To: mingo@elte.hu
Cc: mingo@redhat.com, linux-kernel@vger.kernel.org,
     linux-rt-users@vger.kernel.org
Subject: [PATCH -rt 4/8] fork: desched_thread comment rework.

Lines are too long..

Signed-off-by: Daniel Walker <dwalker@mvista.com>

---
 kernel/fork.c |    6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

Index: linux-2.6.24.4-rt4/kernel/fork.c
===================================================================
--- linux-2.6.24.4-rt4.orig/kernel/fork.c	2008-03-24 19:06:48.000000000 -0400
+++ linux-2.6.24.4-rt4/kernel/fork.c	2008-03-24 19:07:03.000000000 -0400
@@ -1839,8 +1839,10 @@ static int desched_thread(void * __bind_
 			continue;
 		schedule();
 
-		/* This must be called from time to time on ia64, and is a no-op on other archs.
-		 * Used to be in cpu_idle(), but with the new -rt semantics it can't stay there.
+		/*
+ 		 * This must be called from time to time on ia64, and is a
+ 		 * no-op on other archs. Used to be in cpu_idle(), but with
+ 		 * the new -rt semantics it can't stay there.
 		 */
 		check_pgt_cache();
 
